test/visualizer/seaport/index.test.ts

```diff
@@ -0,0 +1,244 @@
+import { Domain } from "../../../src/types";
+import visualizer from "../../../src/visualizer";
+import {
+  seaportBid,
+  seaportBundleListing,
+  seaportCollectionOfferAnyToken,
+  seaportCollectionOfferSpecificTrait,
+  seaportDutchAuction,
+  seaportEnglishAuction,
+  seaportSellMessagePayload,
+} from "./data";
+
+describe("visualizer", () => {
+  describe("seaport v1.4", () => {
+    const seaPortDomainVersion2: Domain = {
+      verifyingContract: "0x00000000006c3852cbEf3e08E8dF289169EdE581",
+      name: "Seaport",
+      version: "1.2",
```

Comment:
You are using v1.1 Seaport address.
If your intention was to use v1.1 I suggest updating the variable to `seaPortDomainVersion1` and also the version variable above

Reply:
yes it looks like i mixed up the versions, I will check which one is more appropriate and update accordingly